korgan/client_document_runtime_guidance.py

```
@@ -0,0 +1,66 @@
+_INSTALLED = False
+
+
+def _wrap_prompt(
+    original: Callable[..., Awaitable[None]],
+    kind: str,
+) -> Callable[..., Awaitable[None]]:
+    """Append one request-atomic checklist after the existing prompt."""
+    if getattr(original, "_korgan_client_guidance", False):
+        return original
+
+    async def guided(*args: Any, **kwargs: Any) -> None:
```

Ruff (0.16.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)